client/my-sites/plugins/plugins-browser-item/index.jsx

```diff
@@ -189,21 +193,39 @@ const InstalledInOrPricing = ( {
 	currentSites,
 } ) => {
 	const translate = useTranslate();
+	const currentSite = currentSites.length === 1 ? currentSites[ 0 ] : null;
+	const installedPlugins =
+		useSelector( ( state ) => getInstalledPlugins( state, [ currentSite?.ID ] ) ) || [];
+	const activePlugins = installedPlugins.filter(
+		( activePlugin ) => activePlugin.sites[ currentSite?.ID ]?.active
+	);
 
 	if ( ( sitesWithPlugin && sitesWithPlugin.length > 0 ) || isWpcomPreinstalled ) {
+		/* eslint-disable wpcalypso/jsx-gridicon-size */
+
+		const isActive = !! activePlugins.find( ( activePlugin ) => activePlugin.slug === plugin.slug );
+
 		return (
-			/* eslint-disable wpcalypso/jsx-gridicon-size */
-			<div className="plugins-browser-item__installed">
-				<Gridicon icon="checkmark" size={ 14 } />
-				{ isWpcomPreinstalled || currentSites.length === 1
-					? translate( 'Installed' )
-					: translate( 'Installed on %d site', 'Installed on %d sites', {
-							args: [ sitesWithPlugin.length ],
-							count: sitesWithPlugin.length,
-					  } ) }
+			<div className="plugins-browser-item__installed-and-active-container">
+				<div className="plugins-browser-item__installed">
+					<Gridicon icon="checkmark" size={ 14 } />
+					{ isWpcomPreinstalled || currentSites?.length === 1
+						? translate( 'Installed' )
+						: translate( 'Installed on %d site', 'Installed on %d sites', {
+								args: [ sitesWithPlugin.length ],
+								count: sitesWithPlugin.length,
+						  } ) }
+				</div>
+				{ currentSites?.length === 1 && (
```

Comment:
```suggestion
				{ selectedSiteId && (
```

should be enough here.